lib/elixir_google_scraper/scraper.ex

```diff
@@ -0,0 +1,38 @@
+defmodule ElixirGoogleScraper.Scraper do
+  @moduledoc """
+  The Scraper context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias ElixirGoogleScraper.Accounts.User
+  alias ElixirGoogleScraper.Repo
+  alias ElixirGoogleScraper.Scraper.CSVKeyword
+  alias ElixirGoogleScraper.Scraper.Keyword
+
+  def create_keyword(attrs \\ %{}) do
```

Comment:
Should we make it private or it will be used publicly later. 🤔 

```suggestion
  defp create_keyword(attrs \\ %{}) do
```